modules/c2c/cloud2murano.lua

```diff
@@ -97,7 +96,7 @@ function cloud2murano.data_in(location, options)
 
   local payload = {{ -- a list
     values = data,
-    timestamp = (options.timestamp or os.time(os.date("!*t")))
+    timestamp = (options.timestamp or os.time(os.date("!*t")) * 1000000)
```

Comment:
Here need to be consistent, with line 28, if we consider timestamp to already have the correct unit.
So remove the '('

Reply:
```suggestion
    timestamp = options.timestamp or os.time(os.date("!*t")) * 1000000
```